docs/manual/docs/administrator-guide/managing-users-and-groups/index.md

```diff
@@ -76,3 +76,126 @@ Rights associated with the roles are illustrated in detail in the list below:
+    | Metadata selection / Publish                |                         |                         | | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Metadata selection / Unpublish              |                         |                         | | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Metadata selection / Approve                |                         |                         | | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Metadata selection / Transfer Ownership     |                         |                         | | | :material-check-circle: | :material-check-circle: |
+    | Metadata selection / Validate               |                         |                         | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Metadata selection / Validate records links |                         |                         | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Metadata selection / Updates categories     |                         |                         | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Metadata selection / Delete                 |                         |                         | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Metadata selection / Index records          |                         |                         | | | :material-check-circle: | :material-check-circle: |
+    | Preferred Records                           | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Watch list                                  | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | "Sorted by relevancy,  modified,  title..." | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+
+=== "Metadata Page"
+ 
+    |                                             | Anonymous user          |     Registered User     | Editor| Reviewer|       User Admin        |     Admini- strator     |
+    |---------------------------------------------| :---: |:-----------------------:| :---: | :---: |:-----------------------:|:-----------------------:|
+    | Edit| | | :material-check-circle: | :material-check-circle: |          EDIT           | :material-check-circle: |
+    | Delete| | | :material-check-circle: | :material-check-circle: |           DEL           | :material-check-circle: |
```

Comment:
For `UserAdmin` I was not able to create or import metadata, not sure if this applies.